Lombiq.Tests.UI/Services/OrchardCoreUITestExecutorConfiguration.cs

```diff
@@ -161,6 +161,9 @@ public void AssertBrowserLogMaybe(IList<BrowserLogMessage> browserLogs, Action<s
                 !message.Message.ContainsOrdinalIgnoreCase("HTML Imports is deprecated") &&
                 // The 404 is because of how browsers automatically request /favicon.ico even if a favicon is declared
                 // to be under a different URL.
-                !message.IsNotFoundMessage("/favicon.ico");
+                !message.IsNotFoundMessage("/favicon.ico") &&
+                // Ignore if browser telemetry is not available.
+                !message.Message.ContainsOrdinalIgnoreCase(
+                    "https://dc.services.visualstudio.com/v2/track - Failed to load resource: the server responded with a status of 400");
```

Comment:
Why do we have this back? As I mentioned before, the AI module should be configured with offline mode, see `UITestBase` in the Samples. Such requests should never happen during UI testing.

Reply:
By the way is there any reason AI isn't offline by default in the UI tests, or there isn't at least an extension method for turning it off?

Reply:
For both we need something related to UI testing in the AI module that we don't have yet but I added it here: https://github.com/Lombiq/Orchard-Azure-Application-Insights/issues/25

Reply:
[Added the method!](https://github.com/Lombiq/UI-Testing-Toolbox/pull/132/commits/e81c679df9eaa1554c50a2e3b743a34c58ab2014#diff-b1534dd6c730fb4e0c76a1498e1f41e0af19b8f6e79d4c9dea5d1906257de8c0R33)

Reply:
Nice!